code/game/objects/items.dm

```diff
@@ -520,10 +520,6 @@
 	add_fingerprint(usr)
 	return ..()
 
-/obj/item/vv_get_dropdown()
-	. = ..()
-	VV_DROPDOWN_OPTION(VV_HK_ADD_FANTASY_AFFIX, "Add Fantasy Affix")
-
```

Comment:
...?

Reply:
I noticed in VV dropdown this line was copied twice so I deleted the dupe